operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/workflow/DependentResourceNode.java

```diff
@@ -58,32 +57,28 @@ public void addDependsOnRelation(DependentResourceNode node) {
 
   @Override
   public String toString() {
-    return "{"
-        + parents.stream().map(p -> p.dependentResource.toString())
-            .collect(Collectors.joining(", ", "[", "]->"))
-        + "(" + dependentResource + ")"
-        + dependsOn.stream().map(d -> d.dependentResource.toString())
-            .collect(Collectors.joining(", ", "->[", "]"))
-        + '}';
+    return "DependentResourceNode{" +
```

Comment:
Why, basically a reverted this, since you can see easily the parents and the whole structure just by looking on workflow definition. So was debugging this, and searching for issues during the development on lot's of occasions, and this was always sufficient to find the problem.